Update go-iptables dependency with bug fix

Encounter this problem when using CNI bridge plugin on K8s. 

```console
panic: interface conversion: error is syscall.Errno, not *iptables.Error

goroutine 1 [running, locked to thread]:
panic(0x5d6bc0, 0xc820058780)
	/home/mixia/.gimme/versions/go1.6.linux.amd64/src/runtime/panic.go:464 +0x3e6
github.com/appc/cni/vendor/github.com/coreos/go-iptables/iptables.(*IPTables).Exists(0xc8200aea60, 0x6135c8, 0x3, 0xc8200a8300, 0xb, 0xc8200bfae8, 0x4, 0x4, 0x4, 0x0, ...)
	/home/mixia/cni/gopath/src/github.com/appc/cni/vendor/github.com/coreos/go-iptables/iptables/iptables.go:89 +0x5c7
github.com/appc/cni/vendor/github.com/coreos/go-iptables/iptables.(*IPTables).AppendUnique(0xc8200aea60, 0x6135c8, 0x3, 0xc8200a8300, 0xb, 0xc8200bfae8, 0x4, 0x4, 0x0, 0x0)
	/home/mixia/cni/gopath/src/github.com/appc/cni/vendor/github.com/coreos/go-iptables/iptables/iptables.go:110 +0x8f
github.com/appc/cni/pkg/ip.SetupIPMasq(0xc8200a2630, 0xc8200a8300, 0xb, 0x0, 0x0)
	/home/mixia/cni/gopath/src/github.com/appc/cni/pkg/ip/ipmasq.go:39 +0x3fd
main.cmdAdd(0xc82004e070, 0x0, 0x0)
	/home/mixia/cni/gopath/src/github.com/appc/cni/plugins/main/bridge/bridge.go:224 +0x6eb
github.com/appc/cni/pkg/skel.PluginMain(0x66a0c8, 0x66a0d0)
	/home/mixia/cni/gopath/src/github.com/appc/cni/pkg/skel/skel.go:86 +0x773
main.main()
	/home/mixia/cni/gopath/src/github.com/appc/cni/plugins/main/bridge/bridge.go:250 +0x2f
```

It turns out the go-iptables client is returning `bad file descriptor` error. 
PR for this in flight: https://github.com/coreos/go-iptables/pull/15

Please update dependency once fix gets in.

--- a/Godeps/Godeps.json
+++ b/Godeps/Godeps.json
@@ -7,7 +7,8 @@
 	"Deps": [
 		{
 			"ImportPath": "github.com/coreos/go-iptables/iptables",
-			"Rev": "90456be57fcb8185b264b77ce42a9539df42df25"
+			"Comment": "v0.1.0",
+			"Rev": "fbb73372b87f6e89951c2b6b31470c2c9d5cfae3"
 		},
 		{
 			"ImportPath": "github.com/coreos/go-systemd/activation",

